### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by pH142857** [MartinPL/Tray-Icons-Reloaded#131]
**Describe the bug**
I installed the extension. I added an app in the parameter (Nextcloud client, if that matters). The app is running. Nothing happens: the tray zone doesn't appear at all.
**Desktop:**
 - Distro: Debian 12
 - GNOME Shell: 43.6
 - Extension version: 26
 - Display server: Wayland

 I read the compatibility note: [✓]

**Comment by bulletbs** [comment]
Same behavior for me on the Gnome 43.6 with Wayland.

**Comment by pH142857** [comment]
Update: the tray works afterall, another app appeared (Ferdium, run by AppImage), even without explicitly being added in the Tray Icons parameters. So it seems to work for some apps but not others, even when they are added in the parameters.

**Comment by andreaxhearts** [comment]
Yeah I'm not able to get it to do anything either. Ubuntu 23.10 / GNOME 45.1 / Wayland.

**Comment by dpicassom** [comment]
same in cachyOS, some work some don't. Dont work: whatsie (qt) nextcloud, firewall. 

work: vlc, discord, 
dont work: qbit torrent, whatsie, nextcloud

**Comment by dpicassom** [comment]
thanks to vnepogodin, the solution to tray icons not appearing in wayland is to set `GDK_BACKEND` to `'x11'`,

test by Command
 `GDK_BACKEND=x11 <APPLICATION>` 
 for me this worked (fish is the default shell):
 
 ```fish
 if [ "$XDG_SESSION_TYPE" = "wayland" ]
	set -gx GDK_BACKEND 'x11,wayland'
end
```

in the fish configuration file

**Comment by artichoked1** [comment]
I'm having this issue too. I installed the extension, but nothing changes in my tray when I try to adjust the settings.

Ubuntu 22.10
GNOME 45.1
Xorg display server
